src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,15 @@
+.catalog {
```

Comment:
place your catalog at the center of your page to pass tests
<img width="1065" alt="Screenshot 2023-05-08 at 21 26 46" src="https://user-images.githubusercontent.com/117587463/236902258-f8a1abbd-9097-41b8-89cc-a3a41e0615c3.png">